q3vcftools/src/au/edu/qimr/vcftools/AmalgamatorGS.java

```diff
@@ -169,7 +169,7 @@ private void writeOutput() throws FileNotFoundException {
 				}
 				String scoreS = score + "/" + (4 + vcfFiles.length);
 				scoreDist.computeIfAbsent(scoreS, v -> new AtomicInteger()).incrementAndGet();
-				ps.println(cp.getChromosome() + Constants.TAB + cp.getStartPosition() + Constants.TAB +cp.getName() + Constants.TAB 
+				ps.println(cp.getChromosome() + Constants.TAB + cp.getStartPosition() + Constants.TAB +cp.getRef() + Constants.TAB
 						+ cp.getAlt() + Constants.TAB + Arrays.stream(p.getLeft()).map(s -> null == s ? "./." : s).collect(Collectors.joining(Constants.TAB_STRING))
 						+ Constants.TAB +  Arrays.stream(p.getRight()).map(s -> null == s ? "." : s).collect(Collectors.joining(Constants.TAB_STRING))
 						+ Constants.TAB +  Arrays.stream(missingACs).map(s -> null == s ? "." : s).collect(Collectors.joining(Constants.TAB_STRING))
```

Comment:
Minor query on consistency — is there a rationale for using `TAB` and `TAB_STRING` from `Constants` but not `SLASH_STRING`, `MISSING_DATA`, `MISSING_GT` etc. for the other string literals here?

Reply:
Complete lack of consistency - using more Constatns now (even using static imports)